documentation/asciidoc/topics/proc_recovering_rolling_upgrades.adoc

```diff
@@ -0,0 +1,23 @@
+[id='recovering_rolling_upgrades_{context}']
+= Recovering from a failed Hot Rod rolling upgrade
+
+[role="_abstract"]
+You can roll back failed Hot Rod rolling upgrade to a previous version if the original cluster is still present.
+
+.Prerequisites
+
+* Hot Rod rolling upgrade is in progress and the initial {brandname} cluster is present.
+
+.Procedure
+. Ensure the Hot Rod rolling upgrade is in progress.
++
+[source,options="nowrap",subs=attributes+]
+----
+{oc_get_infinispan} <cr_name> -o yaml
+----
++
+The `status.hotRodRollingUpgradeStatus` field must be present.
++
+. Update `spec.version` field of your `Infinispan CR` to initiate rollback to the original cluster version.
```

Comment:
We should be more explicit here. We need to update `spec.version` to the original `spec.version` prior to the failed upgrade, this version is recorded in the `status.hotRodRollingUpgradeStatus.SourceVersion` field. An invalid value will result in the Infinispan CR update being rejected by kubernetes.

Reply:
So maybe the following:

"Update the `spec.version` field of your Infinispan CR to the original cluster version defined in `status.hotRodRollingUpgradeStatus.SourceVersion`."